ntirpc/rpc/svc.h

```diff
@@ -139,12 +139,15 @@ typedef struct svc_init_params {
 	u_int gss_max_gc;
 	uint32_t channels;
 	int32_t idle_timeout;
+	uint32_t thr_stack_size;
 } svc_init_params;
 
 /* Svc param flags */
 #define SVC_FLAG_NONE             0x0000
 #define SVC_FLAG_NOREG_XPRTS      0x0001
 
+#define SVC_PARAM_HAS_THR_STACK_SIZE 1
```

Comment:
This appears to be unused?

Reply:
Ah, it's used in Ganesha.

Reply:
It's used in ganesha (and other theoretical clients). Right now it's particularly useful for building with this change before this change propagates out in the wild.

https://review.gerrithub.io/c/ffilz/nfs-ganesha/+/1168839/3/src/MainNFSD/nfs_rpc_dispatcher_thread.c